packages/node/changelog/1.2.1/20230811184244-Changelog item.yml

```diff
@@ -0,0 +1,4 @@
+---
+type: Added
+description: |
+  `CHANGELOG.md` for an better overview of the changes.
```

Comment:
The file name contains a space, is this expected>

Reply:
It's allowed. Yes

Reply:
I renmaed it though

Reply:
I renmaed it though